app/components/rdf-form-fields/error-message-input-field.js

```diff
@@ -0,0 +1,54 @@
+import SimpleInputFieldComponent from '@lblod/ember-submission-form-fields/components/rdf-input-fields/simple-value-input-field';
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import { tracked } from '@glimmer/tracking';
+import { SHACL } from '@lblod/submission-form-helpers';
+import { Statement } from 'rdflib';
+
+export default class ErrorMessageInputFieldComponent extends SimpleInputFieldComponent {
+  inputId = 'select-' + guidFor(this);
+
+  @tracked value;
+
+  constructor() {
+    super(...arguments);
+  }
+
+  @action
+  updateValue(e) {
+    let newErrorMessage = e.target.value.trim();
+    const oldErrorMessage = this.args.formStore.match(
+      this.storeOptions.sourceNode,
+      SHACL('resultMessage'),
+      undefined,
+      this.storeOptions.sourceGraph
+    );
+    if (newErrorMessage.length > 0) {
+      this.value = newErrorMessage;
+
+      if (oldErrorMessage) {
+        this.args.formStore.removeStatements(oldErrorMessage);
+      }
+    } else {
+      const errorMessageInForm = this.args.formStore.any(
+        this.storeOptions.sourceNode,
+        SHACL('resultMessage'),
+        undefined,
+        this.storeOptions.sourceGraph
+      );
+      if (errorMessageInForm) {
+        this.value = errorMessageInForm;
+      }
+    }
```

Comment:
If I write a result message:
1. focus out of input field
2. clear the input field as i dont want it
3. Focus out it of the field
4. It sets back the value that was before

Think that the the else is wrong here